packages/database/src/seed/appeal-documents-dev.js

```diff
@@ -177,6 +177,246 @@ const appealDocuments = [
 		origin: 'pins',
 		stage: 'decision',
 		AppealCase: {}
+	},
+	{
+		id: '620776b8-f92d-4f84-8b7e-59ed8706d49a',
+		filename: 'test-conservation-map.pdf',
+		originalFilename: 'test-conservation-map.pdf',
+		size: 16,
+		mime: 'text/plain',
+		documentURI: '',
+		dateCreated: new Date(Date.now()),
```

Comment:
Why not just `new Date()`?
<img width="473" alt="Screenshot 2024-03-22 at 10 16 35" src="https://github.com/Planning-Inspectorate/appeal-planning-decision/assets/35689158/58a4217a-d566-4ec6-9d88-8f56cf2be675">